modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb

```diff
@@ -22,14 +22,20 @@ def perform(power_of_attorney_id, form_number, rep_id, action)
 
         output_path = pdf_constructor(form_number).construct(data(power_of_attorney, form_number, rep),
                                                              id: power_of_attorney.id)
+        power_of_attorney.form_data.dig('serviceOrganization', 'poaCode')
 
         if Flipper.enabled?(:lighthouse_claims_api_poa_use_bd)
           doc_type = form_number == '2122' ? 'L190' : 'L075'
           benefits_doc_upload(poa: power_of_attorney, pdf_path: output_path, doc_type:, action:)
         else
           upload_to_vbms(power_of_attorney, output_path)
         end
-        ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id, rep)
+
+        if dependent_filing?(power_of_attorney)
+          ClaimsApi::PoaAssignDependentClaimantJob.perform_async(power_of_attorney.id)
+        else
+          ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id, rep)
+        end
```

Comment:
Here and in v1: I wonder if it's worth putting the dependent claimant assignment logic in the `PoaUpdater` 🤔

Reply:
It is my understanding we run one or the other

Reply:
if it means more code to do it the other way, lets just stick with this way.

Reply:
Well not sure about 'more' but personally I hate the idea of melting the two together, I think they are _similar_ but have different responsibilities (veteran/dependent).  I think melting them together could create a kind of if/else nightmare that makes it harder to maintain in the long run.  In my _opinion_ at least